pages/index.js

```diff
@@ -1,83 +1,156 @@
+import Link from "next/link";
+import styled, { keyframes, css } from "styled-components";
 
-export default function Homepage({ toggleFavourite, reminders }) {
-  const { data: plants, error, isLoading, mutate } = useSWR("/api/plants");
-
-  const [showForm, setShowForm] = useState(false);
-  const [filteredPlants, setFilteredPlants] = useState([]);
-
-  useEffect(() => {
-    if (plants) {
-      setFilteredPlants(plants);
-    }
-  }, [plants]);
-
-  function handleToggleForm() {
-    setShowForm(!showForm);
-  }
-
-  async function handleAddPlant(newPlantData) {
-    const response = await fetch("/api/plants", {
-      method: "POST",
-      headers: {
-        "Content-Type": "application/json",
-      },
-      body: JSON.stringify(newPlantData),
-    });
-
-    if (response.ok) {
-      setShowForm(false);
-      mutate();
-    }
-  }
-
-  if (isLoading) return <p>Loading plants...</p>;
-  if (error) return <p>Failed to load plants.</p>;
+const weatherAnimation = "/animation/weather.json";
+const careTipAnimation = "/animation/caretip.json";
+const fixedPlantAnimation = "/animation/plantgrowing.json";
 
+export default function LandingPage() {
   return (
     <>
-      <ButtonContainer>
-        <ButtonAdd onClick={handleToggleForm}>
-          {showForm ? "Hide form" : "Add new plant"}
-        </ButtonAdd>
-      </ButtonContainer>
-
-      {showForm && (
-        <PlantForm
-          onSubmitPlant={handleAddPlant}
-          onToggleForm={handleToggleForm}
-        />
-      )}
+      <Cardcontainer>
+        <Card
+          $animation={css`
+            ${gradientMoveTopLeftToBottomRight}
+          `}
+        >
+          <StyledLink href={"/create"}>
+            <p>Create New Plant</p>
+          </StyledLink>
+        </Card>
+        <Card
+          $animation={css`
+            ${gradientMoveTopToBottom}
+          `}
+        >
+          <StyledLink href={"/plants/"}>
+            <p>My Plant List</p>
+          </StyledLink>
+        </Card>
+        <Card
+          $animation={css`
+            ${gradientMoveTopRightToBottomLeft}
+          `}
+        >
+          <StyledLink href={"/favourites"}>
+            <p>My Owned Plants</p>
+          </StyledLink>
+        </Card>
+      </Cardcontainer>
 
-      <SearchPlant setFilteredPlants={setFilteredPlants} allPlants={plants} />
+      <Cardcontainer>
+        <Weathercard>
+          <StyledWeatherLink href={"/"}>
+            Hold onto your leaves! Weather feature sprouting soon ... 🌤️
+            <div>
+              <Player
+                autoplay
+                loop
+                src={weatherAnimation}
+                style={{ height: "100px", width: "100px" }}
+              />
+            </div>
+          </StyledWeatherLink>
+        </Weathercard>
+      </Cardcontainer>
+      <Cardcontainer>
+        <Caretipcard>
+          <div>
+            <Player
+              autoplay
+              loop
+              src={careTipAnimation}
```

Comment:
Animations are decorative elements and should not interfere with accessibility tools. So consider adding a `aria-hidden `attribute to not confusing screen readers
```suggestion
            <Player
              autoplay
              loop
              aria-hidden = "true"
              src={careTipAnimation}
```